### PR TITLE
Deprecate inner text everywhere

### PR DESCRIPTION
**Pull request by robmen**

### CODE REVIEW

File: src/WixToolset.Core/Compiler.cs [hunk 1]
```diff
@@ -6394,62 +6345,15 @@ private string ParseConditionElement(XElement node, string parentElementLocalNam
...
-                        }
-                        else
-                        {
-                            this.Core.UnexpectedAttribute(node, attrib);
-                        }
-                        break;
-                    case "Message":
-                        if ("Fragment" == parentElementLocalName || "Product" == parentElementLocalName)
-                        {
+                        case "Cndition":
```
Comment (rseanhall) [inline src/WixToolset.Core/Compiler.cs:6348]: typo

File: src/WixToolset.Core/Compiler_UI.cs [hunk 2]
```diff
@@ -1417,10 +1423,9 @@ private void ParseControlElement(XElement node, string dialog, TupleDefinitionTy
                             }
                         }
 
-                        text = Common.GetInnerText(child);
                         if (!String.IsNullOrEmpty(text) && null != sourceFile)
                         {
-                            this.Core.Write(ErrorMessages.IllegalAttributeWithInnerText(childSourceLineNumbers, child.Name.LocalName, "SourceFile"));
+                            this.Core.Write(ErrorMessages.IllegalAttributeWithOtherAttribute(childSourceLineNumbers, child.Name.LocalName, "SourceFile", "Text"));
```
Comment (rseanhall) [inline src/WixToolset.Core/Compiler_UI.cs:1428]: Should be "Value" not "Text".

File: src/WixToolset.Core/Common.cs [hunk 3]
```diff
@@ -690,10 +690,8 @@ internal static YesNoType GetAttributeYesNoValue(IMessaging messaging, SourceLin
         /// <summary>
         /// Gets the text of an XElement.
         /// </summary>
-        /// <param name="sourceLineNumbers">Source line information about the owner element.</param>
-        /// <param name="attribute">The attribute containing the value to get.</param>
-        /// <param name="messageHandler">A delegate that receives error messages.</param>
-        /// <returns>The attribute's YesNoType value.</returns>
+        /// <param name="node">Element to get text.</param>
+        /// <returns>The element's text.</returns>
```
Comment (rseanhall) [inline src/WixToolset.Core/Common.cs:694]: I'm surprised you didn't mark this as obsolete.

File: src/WixToolset.Core/Compiler.cs (hunk 1 above)
Comment (robmen) [reply src/WixToolset.Core/Compiler.cs:6348]: This is what I get for not writing all those tests I knew I should have written but didn't. :cry:

File: src/WixToolset.Core/Compiler_UI.cs (hunk 2 above)
Comment (robmen) [reply src/WixToolset.Core/Compiler_UI.cs:1428]: Thanks.

File: src/WixToolset.Core/Common.cs (hunk 3 above)
Comment (robmen) [reply src/WixToolset.Core/Common.cs:694]: It turns out this method is also used by localization. I think that's what pulled the method out of the Compiler and into Common. Anyway, I haven't thought through the implications of changing .wxls to use attributes over text. So I paused here (but fixed the xmldoc).

**Pull request merged**